package.json

```diff
@@ -94,7 +93,7 @@
     "proxy-middleware": "^0.15.0",
     "react-datepicker": "^1.4.0",
     "react-input-autosize": "^1.1.0",
-    "react-select": "^0.9.1"
+    "react-select": "1.2.1"
```

Comment:
😱

Reply:
I checked in SS and it seems to look fine. @GeReV said API is the same in 1.x